Rakefile

```diff
@@ -24,6 +24,16 @@ namespace :test do
     sh "xcodebuild test -configuration Release -scheme #{SCHEME} -destination '#{DESTINATION}' | xcpretty -c; exit ${PIPESTATUS[0]}"
   end
 
+  desc "Runs the unit tests for Scenarios via swiftpm"
+  task :package do
+    begin
+      cp ".Package.test.swift", "Package.swift"
+      sh "swift test"
+    ensure
+      sh "git checkout HEAD -- Package.swift"
```

Comment:
This is neat. 🙂 

Is the '--' here preferential? Up to this point, I have been checking out files from a particular tree-ish without, which would have this look like:

```
git checkout HEAD Package.swift
```